tests/Feature/SubpageTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Subpage;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+#use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class SubpageTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    /**
+     * "Refresh" the database by deleting everything.
+     *
+     * TODO: Use RefreshDatabase instead of DatabaseTransaction/this method.
+     *
+     * @return void
+     */
+    protected function setUp() : void
+    {
+        parent::setUp();
+        DB::table('subpages')->delete();
+    }
+
+    /**
+     * Test show by id.
+     *
+     * @return void
+     */
+    public function testShowById() : void
+    {
+        $subpage = factory(Subpage::class)->create(['id' => 123]);
+
+        $response = $this->get('/subpages/123');
+        $response->assertStatus(200);
```

Comment:
@abm-devmonk-purple There are also new friendly/readable status assertions in Laravel:
`$response->assertOk()`
`$response->assertNotFound()`

Reply:
Done in e6ac5b1d5fd457b5616405fefd32d856b9624c3a.